app/src/main/java/org/mozilla/fenix/components/PrivateShortcutCreateManager.kt

```diff
@@ -26,18 +26,13 @@ object PrivateShortcutCreateManager {
     fun doesPrivateBrowsingPinnedShortcutExist(context: Context): Boolean {
         return if (SDK_INT >= Build.VERSION_CODES.N_MR1) {
             val pinnedShortcuts = context.getSystemService(ShortcutManager::class.java).pinnedShortcuts
-            if (pinnedShortcuts.any()) {
-                for (s in pinnedShortcuts) {
-                    if (s.intent?.extras?.getString(HomeActivity.OPEN_TO_SEARCH) ==
-                        StartSearchIntentProcessor.PRIVATE_BROWSING_PINNED_SHORTCUT) {
-                        return true
-                    }
-                }
-            }
+            if (pinnedShortcuts.any {
+                    it.intent?.extras?.getString(HomeActivity.OPEN_TO_SEARCH) ==
+                    StartSearchIntentProcessor.PRIVATE_BROWSING_PINNED_SHORTCUT })
+                return true
```

Comment:
`any` already returns a boolean, so it can be returned directly without the if statement.

Reply:
Thanks for the suggestion, I like how much cleaner the code looks.  Took me a little while to realize what you were saying though ;) 

I did another push, so I think we're good to go.